demos/atk-init.php

```diff
@@ -42,19 +42,21 @@
 $app->initLayout($app->stickyGET('layout') ?: 'Maestro');
 
 $layout = $app->layout;
-// Need for phpUnit test only for producing right url.
+// Need for phpunit only for producing right url.
 $layout->name = 'atk_admin';
 $layout->id = $layout->name;
 
 if ($layout instanceof \atk4\ui\Layout\Navigable) {
-    $layout->addMenuItem(['Welcome to Agile Toolkit', 'icon' => 'gift'], ['/demos/index']);
+    $demosUrl = './demos/';
+
+    $layout->addMenuItem(['Welcome to Agile Toolkit', 'icon' => 'gift'], [$demosUrl]);
```

Comment:
the original `index` was a typo?

Reply:
I tested it now... It is horrible idea to append `.php` suffixes somewhere. User should either pass a path or a routing name/path.